lib/sslcommon.c

```diff
@@ -210,7 +215,36 @@ static int gftp_ssl_post_connection_check (gftp_request * request)
                       ok = 1;
                       break;
                     }
-                }
```

Comment:
Why is this code chunk staying around commented out? We can pull the old code from the git historyif needed.